src/elements/checkbox/checkbox.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('semantic.ui.elements.checkbox', [])
+
+    .directive('smCheckbox', smCheckbox);
+
+  function smCheckbox() {
+    return {
+      restrict: 'E',
+      require: '?ngModel',
+      transclude: true,
+      scope: {
+        ngDisabled: '='
+      },
+      template: '<div class="ui checkbox" ng-class="{ checked: isChecked }">' +
```

Comment:
@caitp I was considering avoiding `ng-class` here, so with your vote I'll make that improvement for sure. 

Let me have a bit more of a think about your other suggestions. Thanks for the review :+1: